src/elements/content-sidebar/__tests__/SidebarNavSign.test.tsx

```diff
@@ -1,80 +1,53 @@
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+
+import { userEvent } from '@testing-library/user-event';
+import { render, screen } from '../../../test-utils/testing-library';
 import SidebarNavSign from '../SidebarNavSign';
-// @ts-ignore Module is written in Flow
-import FeatureProvider from '../../common/feature-checking/FeatureProvider';
 
 describe('elements/content-sidebar/SidebarNavSign', () => {
     const onClickRequestSignature = jest.fn();
     const onClickSignMyself = jest.fn();
```

Comment:
nit: usually we create these locally in the individual tests so that they're not reused across tests. otherwise, it's easy to overlook that if the mocks are not cleared properly, then their implementations and call counts would be carried over between tests.